commands/miner.go

```diff
@@ -62,6 +62,25 @@ additional sectors.`,
 	Run: func(req *cmds.Request, re cmds.ResponseEmitter, env cmds.Environment) error {
 		var err error
 
+		pp, err := GetPorcelainAPI(env).ProtocolParameters(env.Context())
+		if err != nil {
+			return err
+		}
+
+		// TODO: Modify this command to accept a sector size as an argument,
+		// ensuring that the provided sector size is supported by the network.
+		// https://github.com/filecoin-project/go-filecoin/issues/2530
+		//
+		// TODO: It may become the case that the protocol does not specify an
```

Comment:
This is contentious and at any rate, it's not really a TODO. I'd remove this todo.